UPGRADE-14.0.md

```diff
@@ -211,6 +211,189 @@ Follow the instructions in relevant sections, e.g. `shopsys/coding-standards` or
 -   upgrade Storefront docker image ([#2931](https://github.com/shopsys/shopsys/pull/2931))
     -   now we use Node.js version 20-alpine3.17 and PNPM version 8.10.5
     -   see #project-base-diff to update your project
+-   handle image resizing by image proxy ([#2924](https://github.com/shopsys/shopsys/pull/2924))
+    -   for local development without CDN, the `imageResizer.php` file has a setup for local imgProxy instance
+    -   if you are using VSH CDN, the image proxy is already set up for you in `imageResizer.php` file.
+    -   for any other CDN, you can use the `imageResizer.php` file as well, but you have to set up the proxy yourself
```

Comment:
shouldn't be mentioned also the nginx.conf file? When someone has more image sizes than the platform, then they need to add a size into a regex so the redirection work properly

Reply:
yes, good point! 6e3ac2c02